apps/opencs/view/world/scriptsubview.cpp

```diff
@@ -198,26 +198,38 @@ void CSVWorld::ScriptSubView::useHint (const std::string& hint)
     if (hint.empty())
         return;
 
-    if (hint[0]=='l')
-    {
-        std::istringstream stream (hint.c_str()+1);
-
-        char ignore;
-        int line;
-        int column;
-
-        if (stream >> ignore >> line >> column)
+    size_t line = 0, column = 0;
+    std::istringstream stream (hint.c_str()+2);
```

Comment:
The error checking is a bit weak here. If hint isn't empty hint.c_str()+1 is guaranteed to be valid. The same can not be said of +2 in a case of an ill-formed hint. Please use the method utilised in the original code.